Library/Homebrew/cmd/update-report.rb

```diff
@@ -115,11 +115,13 @@ def update_report
         puts "No changes to formulae."
       else
         hub.dump
-        hub.reporters.each(&:migrate_tap_migration)
-        hub.reporters.each(&:migrate_formula_rename)
-        CacheStoreDatabase.use(:descriptions) do |db|
-          DescriptionCacheStore.new(db)
-                               .update_from_report!(hub)
+        unless ENV["HOMEBREW_DISABLE_LOAD_FORMULA"]
```

Comment:
Thoughts on doing an early return here instead?

Reply:
Could doing so and thus skip the `Homebrew.failed = true` line be a concern?

Reply:
Good catch. That env is set by `update.sh` so could be moved before the early return.